src/main/java/io/ebean/EbeanServer.java

```diff
@@ -1850,4 +1850,10 @@
    * </p>
    */
   <T> Set<String> validateQuery(Query<T> query);
+
+  /**
+   * Execute the DDL generator manually (this makes only sense if {@link ServerConfig#isAutostart() is false)
+   * @param online
+   */
+  void executeDdlGenerator(boolean online);
```

Comment:
So it seems to me that you'd want to "run migrations" rather than "generate and run the create-all.sql" ?

That is, create-all.sql ... generally aimed for testing and "db migrations" for production.

Reply:
Should this not instead be one of:
- runMigrations()
- start()

Reply:
Yes, runMigrations would be a better naming.

Maybe autostart can be renamed to offline, too. So would be the application's task to create server,  try to run migration and start the server afterwards.

Reply:
Yeah, it is almost `offline` and `startServer()` ... with start server usually meaning run migrations but that depends on ServerConfig. Maybe startServer() .. is a better more generic name.

Bring up EbeanServer "offine" ... then sometime later ... ebeanServer.startServer() ... which maybe runs migrations etc (stuff that is normally done when the server starts).